src/examples/specgan_example/specgan_example.py

```diff
@@ -0,0 +1,274 @@
+import tensorflow as tf
+from tensorflow.compat.v1 import ConfigProto
+from tensorflow.compat.v1 import InteractiveSession
+import matplotlib.pyplot as plt
+
+
+import os
+from ...datasets.nsynth import instruments, nsynth_to_melspec
+from ...models.simple_gan import create_generator, create_discriminator
+from ...datasets import preprocess
+from ...models.losses import create_simple_gan_loss
+from ...models.training import create_gan_train_step
```

Comment:
Relative imports are useful sometimes, but I feel they just obfuscate here.
```suggestion
from datasets.nsynth import instruments, nsynth_to_melspec
from models.simple_gan import create_generator, create_discriminator
from datasets import preprocess
from models.losses import create_simple_gan_loss
from models.training import create_gan_train_step
```

Reply:
Yeah, relative imports doesn't look great

Reply:
My editor complains about importing from parent/parent-parent directories if you don't use relative imports. You are probably right and I also hate them.

As for __init__.py, they are required for relative imports to work, otherwise python does not treat a directory as a module. (some of them might be unnecessary)